packages/liquid_std/liquid_std.0.1/opam

```diff
@@ -0,0 +1,34 @@
+opam-version: "2.0"
+synopsis: "The Standard Libarary for Liquid"
+description: """
+The Standard Libarary for Liquid
+"""
+maintainer: "faerberbendev@protonmail.com"
+authors: ["Ben Faerber"]
+homepage: "https://github.com/benfaerber/liquid-ml"
+bug-reports: "https://github.com/benfaerber/liquid-ml/issues"
+dev-repo: "git+https://github.com/benfaerber/liquid-ml.git"
+license: "MIT"
+depends: [ 
+    "ocaml" { >= "4.11" }
+    "dune" { >= "2.5" } 
+    "base" { >= "0.13.1" }
+    "core" { >= "0.13.0" } 
+    "stdio" { >= "0.13.0" } 
+    "re2" { >= "0.13.0" }
+    "base64" { >= "2.0.0" }
+    "sha" { >= "1.0" }
+    "liquid_syntax" { = "0.1"} 
+    "liquid_parser" { = "0.1"}
+    "liquid_syntax" { = "0.1"} 
```

Comment:
```suggestion
    "liquid_syntax" { = version} 
    "liquid_parser" { = version}
    "liquid_syntax" { = version} 
```

Reply:
Oh I see! I thought you meant for me to change to my version. I will apply the change